### PR TITLE
[bugfix] cybersource - update street2 address

### PR DESCRIPTION
**Pull request by jacoobb**
### Why:
https://chargify.atlassian.net/browse/PGT-77

Problem with second address field update for Cybersource.

### What:
If a customer removes the second address field ActiveMerchand doesn't send this field and this field doesn't update.

In code, the second address is `street2`

_source ActiveMerchant lib/active_merchant/billing/gateways/cyber_source.rb_:
```ruby
def add_address(xml, payment_method, address, options, shipTo = false)
        xml.tag! shipTo ? 'shipTo' : 'billTo' do
          xml.tag! 'firstName',             payment_method.first_name             if payment_method
          xml.tag! 'lastName',              payment_method.last_name              if payment_method
          xml.tag! 'street1',               address[:address1]
          xml.tag! 'street2',               address[:address2]                unless address[:address2].blank?
          [...]
end
```

### Test:

**Steps to replicate:**

1. On a site connected to Cybersource (or this site: https://nschimka-cyber-statement.chargify.com/subscriptions) create a new subscription with all billing address fields filled in. Be sure to include the Address line 2 field.
2. Log into your Cybersource sandbox (or use the ones above which is in 1password).
3. In Cybersource click Transaction Management > Transaction and find the new transaction for the subscription creation and click into it.
4. Take note of the address displayed.
5. Navigate back to the Subscription in Chargify and click “Subscription Actions” and select “View Self-Service Page”.
6. Once on the SSP delete the “Address 2” line and re-enter the test card and click update.
7. Navigate back to the Chargify subscription and use admin functions and process now.
8. Go back into your Cybersource sandbox and notice 3 new transactions for this subscription.
9. Click into the transactions and notice 1 will have the address updated correctly, however all other transactions still display the incorrect address.

### CODE REVIEW

File: lib/active_merchant/billing/gateways/cyber_source.rb [hunk 1]
```diff
@@ -463,7 +463,7 @@ def add_address(xml, payment_method, address, options, shipTo = false)
           xml.tag! 'firstName',             payment_method.first_name             if payment_method
           xml.tag! 'lastName',              payment_method.last_name              if payment_method
           xml.tag! 'street1',               address[:address1]
-          xml.tag! 'street2',               address[:address2]                unless address[:address2].blank?
+          xml.tag! 'street2',               address[:address2]
```
Comment (przemyslawhaptas) [inline lib/active_merchant/billing/gateways/cyber_source.rb:466]: Do you know why this change was introduced in the first place? I was able to track it down to https://github.com/activemerchant/active_merchant/pull/376, which refers https://github.com/activemerchant/active_merchant/pull/329

I'd like to be sure it won't break anything

Comment (przemyslawhaptas) [reply lib/active_merchant/billing/gateways/cyber_source.rb:466]: I can see more similar checks in this method, that's a bit suspicious

Comment (jacoobb) [reply lib/active_merchant/billing/gateways/cyber_source.rb:466]: I recorded again VCR cassettes in https://github.com/chargify/chargify/pull/16701/files with empty street2 and everything works ok.

With `unless` you don't have a chance to remove saved street2 value.
